develop/dev-guide-aws-lambda-integration.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Integrate TiDB Cloud Serverless with Amazon Lambda using AWS CloudFormation
+summary: Introduce how to integrate TiDB with Amazon Lambda and CloudFormation step by step.
+---
+
+# Integrate TiDB Cloud Serverless with Amazon Lambda using AWS CloudFormation
+
+## Introduction
+
+This document provides a step-by-step guide on how to use [AWS CloudFormation](https://aws.amazon.com/cloudformation/) to integrate [TiDB Cloud Serverless](https://www.pingcap.com/tidb-cloud/), a cloud-native distributed SQL database, with Amazon Lambda, a serverless event-driven compute service. Integrating these services allows you to build serverless, event-driven microservices that enhance scalability and cost-effectiveness. AWS CloudFormation automates the creation and management of AWS resources, including Lambda functions, API Gateway, and Secrets Manager.
+
+The document is organized into the following sections:
+
+1. [Prerequisites](#prerequisites)
+2. [Solution Overview](#solution-overview)
+3. [Structure](#structure)
```

Comment:
suggestions:
1. Merge the sections on **Solution Overview** and **Structure** into one.
2. Place the **Prerequisites** section after the merged **Solution Overview** and **Structure** section.